spec/spec_helper.rb

```diff
@@ -11,7 +11,7 @@
 
 # Requires supporting ruby files with custom matchers and macros, etc,
 # in spec/support/ and its subdirectories.
-Dir["#{__dir__}/support/**/*.rb"].sort.each { |f| require f }
+Dir["#{__dir__}/support/**/*.rb"].each { |f| require f }
```

Comment:
This is automatically fixed by https://docs.rubocop.org/rubocop/cops_lint.html#lintredundantdirglobsort